docs/components/motor/_index.md

```diff
@@ -535,22 +538,19 @@ print('Moving: ', moving)
 **Parameters:**
 
 - `ctx` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+- `extra` [(map\[string\]interface{})](https://go.dev/blog/maps): Extra options to pass to the underlying RPC call.
 
 **Returns:**
 
-- [(bool)](https://pkg.go.dev/builtin#bool): True if the motor is currently moving.

```

Comment:
[opt, out of diff] I like the explicitness from the Go tab surrounding bool states:

> True if the motor is currently moving; false if not.

nit: should this (and the Go tab's similar `Returns` language) mirror the `(or attempting to move)` guidance from the intro? 

Maybe, all-together:

> True if the motor is currently moving (or attempting to move); false if not.

Reply:
Ya know, with this copy-paste situation I think the diff is already sufficiently confusing, so I'm not going to touch this in this PR.